### PR TITLE
using redirect.as_view for now

### PR DESCRIPTION
**Pull request by dchukhin**

### CODE REVIEW

File: symposion/urls.py [hunk 1]
```diff
@@ -18,8 +18,9 @@
 urlpatterns = patterns("",
     url(r"^$", RedirectView.as_view(url="/%s/" % URL_PREFIX, permanent=True)),
     url(r"^%s/" % URL_PREFIX, include(patterns("",
-        url(r"^$", TemplateView.as_view(template_name="homepage.html"),
-            name="home"),
+#        url(r"^$", TemplateView.as_view(template_name="homepage.html"),
+#            name="home"),
+        url(r"^$", RedirectView.as_view(url="/2016/sponsors/prospectus/"),  name="home"),
```
Comment (rebecca-caktus) [inline symposion/urls.py]: You'll want to specify `permanent=False` here ([reference](https://docs.djangoproject.com/en/1.8/ref/class-based-views/base/#django.views.generic.base.RedirectView.permanent)).

Comment (rebecca-caktus) [inline symposion/urls.py]: There is also an extra space after the comment.

**Comment by rebecca-caktus** [comment]
:+1:

**Comment by hatlas** [comment]
Passes QA - user is successfully redirected to sponsorship prospectus page from homepage.

**Pull request merged**